includes/class-modal-checkout.php

```diff
@@ -310,6 +312,32 @@ function ( $item ) {
 		}
 	}
 
+	/**
+	 * Process abandon checkout for modal.
+	 */
+	public static function process_abandon_checkout() {
+		if ( is_admin() || ! defined( 'DOING_AJAX' ) ) {
```

Comment:
When doing ajax, `is_admin()` is always true. This check is preventing all `abandon_modal_checkout` requests to get through.

```suggestion
		if ( ! defined( 'DOING_AJAX' ) ) {
```

Reply:
Good catch! Fixed in https://github.com/Automattic/newspack-blocks/pull/1923/commits/8b9930e666cfe35f003bf8ef1c90fb66cd8f01ec